cdap-ui/cypress/integration/pipeline.runtimeargs.spec.ts

```diff
@@ -410,4 +410,19 @@ describe('Deploying pipeline with saved runtime arguments', () => {
     cy.get(dataCy('pipeline-run-btn')).click();
     cy.get(dataCy('Succeeded'), { timeout: PIPELINE_RUN_TIMEOUT }).should('exist');
   });
+
+  it('should have saved runtime arguments available',()=>{
```

Comment:
Can we add more tests to test if we add/edit/remove runtime arguments the same and reopening of the modeless still works correctly?

Reply:
done.